RCS file: /pack/cvsroots/wxwindows/wxWindows/src/common/string.cpp,v
retrieving revision 1.172
diff -r1.172 string.cpp
269a270
>           m_pchData[nLen] = 0;
 

 	  	 
